File: example/scenario/js/types/parallel_shell_commands.js
var noOutputColorConfig = {
    "output": {
        "color": false
    }
};

var command1 = Command
    .config(noOutputColorConfig)
    .value("echo Hello world!");

var command2 = Command
    .value("ps alx | grep java");

Parallel
    .step(command1)
    .step(command2)
    .run();

File: base/src/main/resources/example/scenario/py/types/parallel_shell_commands.py
noOutputColorConfig = {
	"output": {
		"color": False
	}
}

command1 = Command \
	.config(noOutputColorConfig) \
	.value("echo Hello world!") \

command2 = Command \
	.value("ps alx | grep java")

Parallel \
	.step(command1) \
	.step(command2) \
	.run()
